apps/src/templates/sectionAssessments/MultipleChoiceOverviewTable.jsx

```diff
@@ -119,7 +153,13 @@ class MultipleChoiceOverviewTable extends Component {
         transforms: [sortable],
       },
       cell: {
-        props: {style: tableLayoutStyles.cell},
+        props: {
+          style: {
+            ...tableLayoutStyles.cell,
+            ...styles.questionCell,
+            maxWidth: styleConstants['content-width'] - (numAnswers * (ANSWER_COLUMN_WIDTH + PADDING * 2)),
```

Comment:
As far as I can tell, this is the only place you use the PADDING constant.  Why not set it to 20 initially so you don't have to do math?

Reply:
Good point!